recipes/tmap/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "tmap" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/reymond-group/tmap/archive/9aa043dd1af022161048ee0a4b5554ec5e34977f.zip
+  sha256: 56cf4f83d663950ab7ef41deb425b8e1c01de378fb5239aeef7b6613a6edf212
+
+  patches:
+    - 01_bind2d_deprecated.diff
+    - 02_setup_windows.diff
+
+build:
+  number: 0
+  skip: True  # [win]
+  binary_relocation: False  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python {{ python }}
+    - cmake >=3.1
+    - make  # [not win]
+    - ninja  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - openmp  # [win]
+    - setuptools
+    - pybind11
+    - ogdf ==2020.02
+  host:
+    - vc  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
```

Comment:
```suggestion
    - ninja  # [win]
    - setuptools
    - pybind11
    - ogdf 2020.02.*
```

Reply:
(`ninja` should actually be in `build`, but it currently still depends on `python` which we don't want to get in `build`. There is an open PR on the `ninja` feedstock to fix it, so this wouldn't be needed in the future.)